### ISSUE TITLE
Generates invalid ts file if a table has column named "default" 

### ISSUE THREAD

**Issue by msuntharesan** [SweetIQ/schemats#90]
Hello,
I noticed this on postgres.
Ex: 

![image](https://user-images.githubusercontent.com/1296183/38993964-12b47754-43b3-11e8-82c9-a55d60974df4.png)

I think it should rename it to `_default`

**Comment by abenhamdine** [comment]
Indeed, we should have a list of words allowed in SQL, but reserved in Typescript in order to rename them at generation.

You could have used `var` as a column name too 😉

**Comment by xiamx** [comment]
You can modify https://github.com/SweetIQ/schemats/blob/master/src/typescript.ts#L11 and add "default" into the reserved words and make a pull request.

**Comment by abenhamdine** [comment]
> You can modify /src/typescript.ts@master#L11 and add "default" into the reserved words and make a pull request.

Oh nice, didn't know.
